src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -136,6 +143,14 @@ public void create(final EventTypeBase eventType) throws TopicCreationException,
             }
             throw e;
         }
+        try {
+            nakadiKpiPublisher.publish(etLogEventType, () -> new JSONObject()
+                    .put("event_type", eventType.getName())
+                    .put("status", "created")
+                    .put("category", eventType.getCategory().toString()));
```

Comment:
There is no need to call toString(), it will be the same as it is defined in the enum: `UNDEFINED` `DATA` `BUSINESS`

Reply:
ok, thanks. Makes sense. Changed here https://github.com/zalando/nakadi/pull/799/commits/70d110aa249bc846a043d55816abf6451ba6deda